analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -193,8 +193,8 @@ public static class Extensions
         validator.ValidateContainsOperation(OperationKind.Invocation);
         validator.ValidateTag("BeforeField", x => x.HasConstraint(ObjectConstraint.Null).Should().BeTrue());
         validator.ValidateTag("BeforeStaticField", x => x.HasConstraint(ObjectConstraint.Null).Should().BeTrue());
-        validator.ValidateTag("AfterField", x => x.Constraint<ObjectConstraint>().Should().BeNull());
-        validator.ValidateTag("AfterStaticField", x => x.Constraint<ObjectConstraint>().Should().BeNull());
+        validator.ValidateTag("AfterField", x => x.Should().BeNull());
+        validator.ValidateTag("AfterStaticField", x => x.Should().BeNull());
```

Comment:
Clean as you code: This method should be refactored to use `TagValue` instead. The two will end up as `.TagValue("AfterField").Should().HaveNoConstraint()`, effectively ignoring the `null` implementation detail (this UT doesn't care)

Reply:
Same on few methods below - only those you've modified